prometheus-rules/etcd-rules.yaml

```diff
@@ -0,0 +1,117 @@
+# Rules for alerting against etcd, without the etcd alerts: etcdHighNumberOfFailedGRPCRequests
+# Which at the moment are false positives at a rate of 100%
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  generation: 1
+  labels:
+    app: prometheus-operator
+    release: prometheus-kubeaddons
+  name: prometheus-kubeaddons-prom-etcd
+spec:
+  groups:
+  - name: etcd
+    rules:
+    - alert: etcdInsufficientMembers
+      annotations:
+        message: 'etcd cluster "{{ $labels.job }}": insufficient members ({{ $value
+          }}).'
```

Comment:
Are we trying to be strict about whitespace here? I feel this can just as well be a single line? :thinking: